### PR TITLE
https://github.com/open-cluster-management/backlog/issues/16821 --cha…

### PR DESCRIPTION
**Pull request by swopebe**
…nges for app 2 known issues after one was deleted, need more info.

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -564,9 +564,18 @@ Creating more than one channel in the same namespace can cause errors with the h
...
 
 [#ansible-automation-platform-fail]
-=== Ansible Automation Platform (early access) 2.0.0 job fail
+=== Ansible Automation Platform (early access) 2.0.1 and later job fail
 
-When the Ansible Automation Platform (early access) 2.0.0 is installed, `AnsibleJobs` fails to run. If you want to submit prehook and posthook `AnsibleJobs` through {product-title-short}, use the Ansible Automation Platform Resource Operator 0.1.1.
+When the Ansible Automation Platform (early access) 2.0.1 and later is installed, `AnsibleJobs` fails to run. If you want to submit prehook and posthook `AnsibleJobs` through {product-title-short}, use the Ansible Automation Platform Resource Operator 0.1.1.
+
+Use the `cluster-scoped` option to resolve the issue.
+//Please clarify what this option is, the issue does not have anymore details than this.
```
Comment (swopebe) [inline release_notes/known_issues.adoc]: please see comment on line 572

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -564,9 +564,18 @@ Creating more than one channel in the same namespace can cause errors with the h
...
+
+[#ansible-automation-platform-fail]
+=== Ansible Automation Platform operator access Ansible tower outside of a proxy
+
+The Ansible Automation Platform (AAP) operator cannot access Ansible tower outside of a proxy-enabled {ocp-short} cluster. To resolve, you can install the Ansible tower within the proxy.
+//Does user need steps for this?
```
Comment (swopebe) [inline release_notes/known_issues.adoc:575]: please see comment on 578

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (xiangjingli) [reply release_notes/known_issues.adoc]: Here is the draft for clarifying the known issue:

=== Ansible Automation Platform (early access) job fail

When the Ansible Automation Platform (early access) is installed, `AnsibleJobs` fails to run. If you want to submit prehook and posthook `AnsibleJobs` through {product-title-short},  use the `early-access-cluster-scoped` option to resolve the issue. The options is available in the Ansible Automation Platform (early access) version 2.0.1+0.1635279521 and later.

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (xiangjingli) [reply release_notes/known_issues.adoc:575]: The install steps are the same. The only difference is to install the ansible tower on a server, which the ACM hub cluster doesn't need to connect through the proxy server.  The install steps are provided by the ansible tower.

**Review by xiangjingli — APPROVED** [review]
looks good to me

**Pull request merged**